src/homeWork9/ArrayValueCalculator.java

```diff
@@ -0,0 +1,32 @@
+package homeWork9;
+
+public class ArrayValueCalculator {
+
+    public int doCalc(String[][] array) {
+
+        int result = 0;
```

Comment:
Would be better to declare these variables `result, rowNumber, columnNumber` after the validation. So you validate an input first, and then, if all is good, you can declare them and allocate no memory if the input is incorrect.